microfs.py

```diff
@@ -56,10 +56,16 @@ def raw_on(serial):
     """
     Puts the device into raw mode.
     """
-    serial.write(b'\x03')  # Send CTRL-C to break out of loop.
-    serial.read_until(b'\n>')  # Flush buffer until prompt.
-    serial.write(b'\x01')  # Go into raw mode.
-    serial.read_until(b'\r\n>OK')  # Flush buffer until raw mode prompt.
```

Comment:
MicroPython doesn't seem to sending the last `OK`, this is what I get:
```
\r\nraw REPL; CTRL-B to exit\r\n>
```
In this case `read_until` worked because it also returns when no more characters are received.